### ISSUE TITLE
1.14 Support

### ISSUE THREAD

**Issue by TheJellly** [RoboTricker/Transport-Pipes#55]
Plugin doesn't load on a 1.14 spigot server.

**Comment by RoboTricker** [comment]
Yeah, I'm waiting for a dependent library to update, hopefully that happens soon

**Comment by lighth7015** [comment]
What library?

**Comment by DilanRona** [comment]
Probably ProtocolLib

**Comment by lighth7015** [comment]
Oh, it's just waiting for it to come out of "development", then?

**Comment by skerit** [comment]
@RoboTricker The last dev build of ProtocolLib happily runs on 1.14.4, anything we can help you with?

**Comment by sgdc3** [comment]
I'm working on an initial 1.14 update, more news soon

**Comment by MechanicalDragon0687** [comment]
this plugin looks great and would love to see it on 1.14 before 1.15 comes out.
I don't mean to rush (at all), but would like to know if we are looking to see this before christmas?
